ChocChipCookies.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Chocolate Chip Cookies Recipe</title>
+  </head>
+  <body>
+    <h1>Chocolate Chip Cookies</h1>
+    <br>Recipe from:<a href="https://www.kingarthurbaking.com/recipes/chocolate-chip-cookies-recipe">King Arthur Baking</a>
```

Comment:
I think it might be advisable to put the content in a p or h tag or the like. Especially after a br I would also put this text into a new line